crates/node/types/src/lib.rs

```diff
@@ -176,3 +176,64 @@ where
 {
     type Engine = E;
 }
+
+/// Size in bytes.
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
+pub struct ByteSize(pub usize);
```

Comment:
any reason to have this here and not in `crates/node/core/src/args/database.rs` where it's used? I think it can be moved to remove the dependency on `reth-node-types` in `reth-node-core`.

Reply:
Good call, done.